database/migrations/2021_04_12_034131_word_choice.php

```diff
@@ -0,0 +1,52 @@
+<?php
```

Comment:
Migration file name should be create_word_choices_table

Format: `action_<any_descriptive_text>_table`